### ISSUE TITLE
EmptyLineSeparator doesn't require exactly one line between members

### ISSUE THREAD

**Issue by mkordas** [checkstyle/checkstyle#530]
According to [Google Java Style - Vertical Whitespace](http://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.6.1-vertical-whitespace) there must be only one line between class members. Current check doesn't enforce that, so it's incomplete. Please look at samples:

```java
package com.sample;
import java.util.List; // as expected: 'import' should be separated from previous statement
```

```java
package com.sample;

import java.util.List; // OK
```

```java
package com.sample;


import java.util.List; // nothing reported, but violates Vertical Whitespace rules
```

**Comment by romani** [comment]
will be in 6.3

**Comment by romani** [comment]
problem is detected by google style test project - https://travis-ci.org/checkstyle/google-style-config-test/jobs/49054029
